审核 `GBC 2022 Autumn` 规则书 (#4)
RushFTK/osu-wiki: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

## Ruleset

### Ruleset Changes from last tournament
1. Qualifier stage added, for deteriment captain role, result will be used in Drafting phase.

> 2. Player not being picked on Drafting phase will be eliminated.

Author: ```suggestion
2. Player not being picked on Drafting phase will be eliminated.
2. Map pools increased, from `best of X+2` to `best of X+4`
```

Author: 标号等PR合并了再改

The following rules are changed to consistent with rules on `MWC 4k 2022` : 

3. Failed Player's score still get added to the team score.
4. Allowed Team to rematch if any team member disconnected on early of the beatmaps.
5. Referees have rights to lower or higher timer settings.
6. Host have rights to request VOD at any point.
7. Called tactical timeout on Tie-breaker need both teams' captains agreement.
8. Referee **nolonger** have right to temporary modify rules on matches they responsible for.
9. NoFail mod could be choose in FreeMod bracket.

### Tournament rules

1. The GB Cup 2022 Autumn is a captain draft-based team tournament, played on the osu!mania game mode.
   - **While this competition is planned as a 3 versus 3 setup, this might change depending on the number of incoming registrations.**
2. Beatmap scoring is based on Score V2.

> 3. The maps for each round will be announced by the mappool selectors on the Sunday before the actual matches take place.

Author: ```suggestion
3. The maps for each round will be announced by the mappool selectors on the official stream, on the Sunday before the actual matches take place.
```

> 4. Match scheduling will be handled by the host.

Author: ```suggestion
4. The match schedules for each round will be announced by the host on this page, as well as the information sheet, on the Sunday before the matches take place.
```

    - Screenshots from other players taken directly in-game that show the affected player's score.
      - Screenshots from the results screen must clearly show the affected player's score. This is the preferred method.
      - Screenshots taken in-game at the time of disconnection may be accepted. Note that this method does not provide a one-to-one representation of that player's score. Using this method is not encouraged and it may be denied at the referee's discretion if the information provided is not sufficient to identify the player/score.
      - All screenshots **MUST** be taken using the game itself (using `Shift` + `F12`), that is, they must be hosted on the `https://osu.ppy.sh/` domain. Any other form of screenshot will be denied.
     - Player scores may be derived from the official stream as a last resort, in cases where the match is streamed.
11. If less than the minimum amount of required players are present at match time,  the match can be postponed for up to 10 minutes. If, after this period, there are still not enough players for either, a *win by default* will be declared for the team with the most members present.

>     - The minimum amount of required players is defined as the amount of players needed to play a beatmap.

Author: ```suggestion
    - The minimum amount of required players is the amount of players needed to play a beatmap without any vacant spots in the lobby (i.e. 3 participants must be present for the match to begin).
```

    - Disqualification from current and future official and community tournaments, until appealed.
20. Referees may allow, at their discretion, lower or higher tolerances for timers.
21. The host may request liveplays or recordings of individual players or teams at any point.
22. The host reserves the right to modify these rules at any moment. Any such changes will be announced in advance.

### Tournament registration

<!--报名方式是否需要更改？比如向任意选手私聊，汇总，成功报名给出私信等等-->
1. Please send the host a **private message** and **fill [this form](https://wj.qq.com/s2/9902248/95cf/)** to register. 
2. To ensure valid and serious registrations, every registered user will be manually checked by the host.
   - To be successfully registered, a player must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
   - **There are no rank limitations for this tournament.** If there are too many registrants, some of the high-level registrants may be removed from the main tournament and be invited to participate in the showmatch instead.
3. A list of all successfully registered players will be published after the registration phase, and will then participate in the Drafting phase.

> 4. Tournament staff members must not participate as players in the tournament.

Author: ```suggestion
4. [Tournament staff members](/wiki/en/Tournaments/Official_support#what-is-considered-staff) must not participate as players in the tournament, with the exception of commentators and streamers.
```

5. Organisers reserve the right to decline restricted players from participating in the tournament.

### Qualifier instructions

1. In the Qualifier stage, all players will play a specific pool designed by the mappool selectors.

> 2. The Qualifier pool contains 8 maps, all of which will use [FreeMod](/wiki/Game_modifier#freemod) rules.

Author: ```suggestion
2. The Qualifier pool contains 7 maps, all of which will use [FreeMod](/wiki/Game_modifier#freemod) rules,  organised as such:
   - Rice: 3 beatmaps
   - LN: 2 beatmaps
   - Hybrid: 1 beatmap
   - SV: 1 beatmap
```

3. Players will play the mappool at least once at a designated time. Their best combined score will be the final result.
   - Best combined score means totals of each beatmaps best score.

> 4. The mappool will be played in the order listed above.

Author: ```suggestion
4. The mappool will be played in the order listed above.
5. The mappool is to be played in the following order: SV1, RC1, HB1, LN1, LN2, RC2, RC3. Players may not ask to play the pool in any other order.
   -  Players may skip playing any beatmap on their second run.
```

     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
   - Repeat offenders may receive further sanctions from the host.
   - Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
     - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.

### Mappool instructions

1. There will be three separate mappools.
   - Regular stage Round 1 and Round 2 use mappool #1.
   - Regular stage Round 3 and Quarterfinals use mappool #2.
   - Semifinals and Finals use mappool #3.
2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
3. The mappool sizes are as follows:

>    - Qualifiers: 8 beatmaps   

Author: ```suggestion
   - Qualifiers: 7 beatmaps   
```

>    - Mappools #1 & #2: 13 beatmaps

Author: ```suggestion
   - Mappools #1 & #2: 15 beatmaps
```

>    - mappool #3: 15 beatmaps.

Author: ```suggestion
   - mappool #3: 17 beatmaps.
```
